db/migrate/20190731114016_change_importer_and_exporter_to_polymorphic.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeImporterAndExporterToPolymorphic < ActiveRecord::Migration[5.1]
+  def change
+    remove_reference :bulkrax_entries, :importer
+    add_reference :bulkrax_entries, :importerexporter, polymorphic: true, index: {:name => "index_bulkrax_entries_on_importerexporter_type_and_id"}
```

Comment:
this is disassociates all existing entries. can we deconstruct the helpers to make it less so? I didn't look up the actual syntax, but I think it might be something like:

```
remove_index :bulkrax_entries_importer_id
rename_column :bulkrax_entries, :importer_id, :importerexporter_id
add_column :bulkrax_entries, :importerexporter_type, :string, default: 'Bulkrax::Importer'
add_index: :bulkrax_entries, [:importerexporter_id, :importerexporter_type], name: "index_bulkrax_entries_on_importerexporter_type_and_id"
```